src/Discord/Parts/Interactions/Entitlement.php

```diff
@@ -0,0 +1,126 @@
+<?php
+
+/*
+ * This file is a part of the DiscordPHP project.
+ *
+ * Copyright (c) 2015-present David Cole <david.cole1340@gmail.com>
+ *
+ * This file is subject to the MIT license that is bundled
+ * with this source code in the LICENSE.md file.
+ */
+
+namespace Discord\Parts\Interactions;
```

Comment:
I don't think this is part of Interactions

Shouldn't this be part of Monetization (as per Discord's doc resource) or Application (as per the endpoint)?